### ISSUE TITLE
Dashboard/Metrics info documentation

### ISSUE THREAD

**Issue by 2020testuser** [linki/chaoskube#217]
Hi,
I'm looking for documentation to view the chaos pod delete report either in text form or in a dashboard/metrics form. Could you please let me know what args I need to pass and how to use it? Thanks!

**Comment by linki** [comment]
There's a counter of the number of deleted pods exposed via Prometheus under `http://localhost:8080/metrics` by default.

The metric is called `chaoskube_pods_deleted_total`.

You can find other metrics listed [here](https://github.com/linki/chaoskube/blob/master/metrics/metrics.go#L9-L32).

**Comment by 2020testuser** [comment]
@linki - Thanks! 
I've chaoskube deployed using your sample Deployment (with some tweaks)  in my Kubernetes cluster. Could you please let me know how I can see the metrics? 

In your url - http://localhost:8080/metrics, I replaced localhost with my <hostName> and it didn't display the metrics. Do I need to perform any additional step? Sorry. I'm using chaoskube for the first time and so not clear. Thanks!

**Comment by linki** [comment]
No, if you use the Pod IP instead of localhost it should show you something.

**Comment by linki** [comment]
Where do you run the curl command from?

**Comment by 2020testuser** [comment]
Thanks Much! 
I want to run the curl command from my script which I use for chaos test.
Basically, I want to use the curl command to start chaoskube and pass the arguments to kill the pods.
It would be very helpful if you could provide details.

Thanks!
